nixos/modules/services/torrent/transmission.nix

```diff
@@ -35,17 +30,41 @@ in
           Transmission daemon can be controlled via the RPC interface using
           transmission-remote or the WebUI (http://localhost:9091/ by default).
 
-          Torrents are downloaded to ${homeDir}/Downloads/ by default and are
+          Torrents are downloaded to ${cfg.homeDir}/Downloads/ by default and are
```

Comment:
Making option descriptions depend on option values is a bad idea (since it causes the manual to be rebuilt whenever the option value changes).